lib/node_modules/@stdlib/namespace/alias2pkg/data/data.csv

```diff
@@ -109,6 +109,7 @@ base.csub,"@stdlib/math/base/complex/subtract"
 base.dasum,"@stdlib/blas/base/dasum"
 base.daxpy,"@stdlib/blas/base/daxpy"
 base.dcopy,"@stdlib/blas/base/dcopy"
+base.ddot,"@stdlib/blas/base/ddot"
```

Comment:
Thanks for updating these files. Just out of curiosity, how are you updating these database files? I am hoping not manually. :)

Reply:
What kind of programmer would I be if I did things manually? :-D

I am using `make repl-docs` after I edit `@stdlib/namespace/lib/namespace/b.js`.  I was a little frustrated that I couldn't see my `repl.txt` docs in the REPL, so I went fishing in the repo.  I first found all the `build.js` files in `namespace` which led me to `make repl-docs`.

Reply:
Nice! Excellent sleuthing to discover `make repl-docs`! :)